<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContact</class>
 <widget class="QDialog" name="AddContact">
  <property name="windowTitle">
   <string>Add contact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="contactLabelText"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="contactLabelLabel">
     <property name="text">
      <string>Label</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="paymentCodeLabel">
     <property name="text">
      <string>Payment code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="paymentCodeText"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
